services/db-backup/entrypoint.sh

```
@@ -0,0 +1,48 @@
+fi
+
+echo "[entrypoint] starting crond"
+crond -f -l 8 &
+CROND_PID=$!
+
+# Tail the log so `docker logs db-backup` shows backup output
+tail -F "$LOG_FILE" &
+TAIL_PID=$!
+
+# Forward signals to crond for clean shutdown
+trap "kill -TERM $CROND_PID $TAIL_PID 2>/dev/null; exit 0" TERM INT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)